swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -107,7 +107,11 @@ public final class Store: ObservableObject {
       try await self?.handleVPNStatusChange(newVPNStatus: status)
     }
     try ipcClient().subscribeToVPNStatusUpdates(handler: vpnStatusChangeHandler)
-    self.vpnStatus = try ipcClient().sessionStatus()
+
+    let initialStatus = try ipcClient().sessionStatus()
+    self.vpnStatus = initialStatus
+    // Handle initial status to ensure resources start loading if already connected
+    try await handleVPNStatusChange(newVPNStatus: initialStatus)
```

Comment:
```suggestion
    self.vpnStatus = try ipcClient().sessionStatus()
    // Handle initial status to ensure resources start loading if already connected
    try await handleVPNStatusChange(newVPNStatus: self.vpnStatus)
```

Wouldn't this be cleaner?

Reply:
not sure about this, because self.vpnStatus is optional and would have to be unwrapped!

This is hard to spot, as Swift is implicit about the assignment `self.vpnStatus = initialStatus` - in Rust this would have been much more clear, something like: `self.vpnStatus = Some(initialStatus)`

I'd rather keep it as is than to add a needless unwrap that could get triggered if anyone removes the update above.

Reply:
Ah yeah, fair enough. With that knowledge, I agree the temporary variable is cleaner.